### PR TITLE
adding sysdig cloud

### PR DESCRIPTION
**Pull request by 03cranec**
take 2 on this: https://github.com/rancher/community-catalog/pull/304

### CODE REVIEW

File: kubernetes-templates/sysdig-cloud/config.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+name: Sysdig Cloud
+description: |
+  Container-Native Application and Infrastructure Monitoring
+version: 0.1
```
Comment (cloudnautique) [inline kubernetes-templates/sysdig-cloud/config.yml]: Put this in quotes. This is causing it to fail ci.

File: kubernetes-templates/sysdig-cloud/0/rancher-compose.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+.catalog:
+  name: "Sysdig Cloud"
+  version: "0.1"
+  description: "Container-Native Application and Infrastructure Monitoring"
+  questions:
+    - variable: "SDC_ACCESS_KEY"
+      label: "Sysdig Cloud access key"
+      description: "Your unique Sysdig Cloud access key - register for a Sysdig Cloud account at www.sysdig.com to receive a key."
+      type: "string"
```
Comment (cloudnautique) [inline kubernetes-templates/sysdig-cloud/0/rancher-compose.yml]: make this `type: "password"`

File: kubernetes-templates/sysdig-cloud/config.yml (hunk 1 above)
Comment (03cranec) [inline kubernetes-templates/sysdig-cloud/config.yml]: done

File: kubernetes-templates/sysdig-cloud/0/rancher-compose.yml (hunk 2 above)
Comment (03cranec) [inline kubernetes-templates/sysdig-cloud/0/rancher-compose.yml]: done

Comment (cloudnautique) [inline kubernetes-templates/sysdig-cloud/0/rancher-compose.yml]: Sorry, I meant the version string

Comment (03cranec) [inline kubernetes-templates/sysdig-cloud/0/rancher-compose.yml]: aah, yep got it. done.

**Pull request merged**